templates/kubernetes/compare.html

```diff
@@ -425,6 +426,7 @@ <h4>Case study</h4>
         </div>
       </div>
       <p class="p-heading--4"><a href="/engage/atresmedia-charmed-kubernetes-case-study">Atresmedia dominates Spain's OTT media market with Charmed Kubernetes</a></p>
+      <p><a class="p-link--inverted" href="/kubernetes/resources">Find more resources&nbsp;&rsaquo;</a></p>
```

Comment:
```suggestion
      <p><a href="/kubernetes/resources">Find more resources&nbsp;&rsaquo;</a></p>
```
The background is not suru so `p-link--inverted` is not needed